sdk/storage/Azure.Storage.Common/src/Shared/StructuredMessage.cs

```diff
@@ -18,7 +17,7 @@ internal static class StructuredMessage
     public enum Flags
     {
         None = 0,
-        CrcSegment = 1,
+        Crc = 1,
```

Comment:
Sorry I didn't notice this in the first review, but do we maybe want to call this `Crc64`? I could envision a world where we have to add another crc algorithm in the future (like crc32c or something).

Reply:
you're right. I'll actually just match name to the existing content validation enum we have.